ui/tests/integration/components/pki/pki-tidy-form-test.js

```diff
@@ -291,6 +291,7 @@ module('Integration | Component | pki tidy form', function (hooks) {
         { tidy_acme: false },
         'response contains manual tidy params'
       );
+      return { id: 'pki-manual-tidy' };
```

Comment:
My gut tells me I need to add primaryKey = backend to the pik/tidy serializer because the flow of the id being set is off. However, this breaks a lot of other things. .... 🧐